diff -ur cmake-3.18.2.orig/Modules/FindBLAS.cmake cmake-3.18.2/Modules/FindBLAS.cmake
--- cmake-3.18.2.orig/Modules/FindBLAS.cmake	2020-08-20 14:20:32.000000000 +0200
+++ cmake-3.18.2/Modules/FindBLAS.cmake	2020-09-14 19:36:13.952349073 +0200
@@ -27,6 +27,7 @@
   possibilities.  List of vendors valid in this module:
 
   * ``Goto``
+  * ``FlexiBLAS``
   * ``OpenBLAS``
   * ``FLAME``
   * ``ATLAS PhiPACK``
@@ -538,6 +539,22 @@
       ""
       ""
       ""
+      )
+  endif()
+endif()
+
+# FlexiBLAS? (http://www.mpi-magdeburg.mpg.de/mpcsc/software/FlexiBLAS/)
+if(BLA_VENDOR STREQUAL "FlexiBLAS" OR BLA_VENDOR STREQUAL "All")
+  if(NOT BLAS_LIBRARIES)
+    check_blas_libraries(
+      BLAS_LIBRARIES
+      BLAS
+      sgemm
+      ""
+      "flexiblas"
+      ""
+      ""
+      ""
       )
   endif()
 endif()
diff -ur cmake-3.18.2.orig/Modules/FindLAPACK.cmake cmake-3.18.2/Modules/FindLAPACK.cmake
--- cmake-3.18.2.orig/Modules/FindLAPACK.cmake	2020-08-20 14:20:32.000000000 +0200
+++ cmake-3.18.2/Modules/FindLAPACK.cmake	2020-09-14 19:38:43.204639078 +0200
@@ -26,6 +26,7 @@
   If set, checks only the specified vendor, if not set checks all the
   possibilities.  List of vendors valid in this module:
 
+  * ``FlexiBLAS``
   * ``OpenBLAS``
   * ``FLAME``
   * ``Intel10_32`` (intel mkl v10 32 bit)
@@ -349,6 +350,23 @@
         ""
         ""
         ""
+        "${BLAS_LIBRARIES}"
+      )
+    endif()
+  endif()
+
+  # FlexiBLAS? (http://www.mpi-magdeburg.mpg.de/mpcsc/software/FlexiBLAS/)
+  if(BLA_VENDOR STREQUAL "FlexiBLAS" OR BLA_VENDOR STREQUAL "All")
+    if(NOT LAPACK_LIBRARIES)
+      check_lapack_libraries(
+        LAPACK_LIBRARIES
+        LAPACK
+        cheev
+        ""
+        "flexiblas"
+        ""
+        ""
+        ""
         "${BLAS_LIBRARIES}"
       )
     endif()
